portal/templates/flask_user/emails/eproms_password_changed_message.html

```diff
@@ -3,8 +3,20 @@
 {% block message %}
 <p>Your password has been changed.</p>
 {% if user_manager.enable_forgot_password %}
+
+{% if user.organizations.count() == 1 %}
+<p>If you did not initiate this password change, <a href="{{ url_for('user.forgot_password', _external=True) }}">please click here to reset it</a>, or contact your representative at {{ user.organizations.first().name }}.</p>
+{% else %}
 <p>If you did not initiate this password change, <a href="{{ url_for('user.forgot_password', _external=True) }}">please click here to reset it</a>, or contact your representative at {{ app_name }}.</p>
+{% endif %}
+
+{% else %}
```

Comment:
@pbugni added indentations to html template to improve readability (couldn't add them to the txt template, as those would have shown up in the actual email)